pdp/static/js/gridded_observations_map.js

```diff
@@ -94,6 +94,7 @@ function init_obs_map() {
               min = 1;
               max = data.max;
               newParams["STYLES"] = "default/blueheat";
+              newParams["BELOWMINCOLOR"] = 'extend';
```

Comment:
Curious why this was updated too. `gridded_observations_map` doesn't seem to be part of this portal. Guessing it was just to spread the goodness while you were thinking of it.